### DISCUSSION TITLE
who is the sus imposter

### DISCUSSION THREAD

**Discussion by davwat** [CluckCluckChicken/Magnifier#19] [Q&A]
??/?//?//??///?/

**Comment by JackyBorderCollie** [comment]
red is sus

**Comment by FunctionalMetatable** [comment]
more likely, [this](https://functionalmetatable.github.io/miscallaneous/epic-gamers-only/sus) is sus.

**Reply by JackyBorderCollie** [reply]
yes it's extremely sus

**Reply by davwat** [reply]
the fact that it doesnt have autoplay is even more susssss
